src/network/subscription.rs

```diff
@@ -32,15 +32,16 @@ pub fn process_gossip<S>(inbound: S, state: GlobalStateR) -> tokio::executor::Sp
 where
     S: Stream<Item = Gossip<Node>, Error = core_error::Error> + Send + 'static,
 {
+    let err_logger = state.logger().clone();
```

Comment:
Now this function could receive the logger from the caller, similarly to `process_block_announcements`. Both server and client tasks can use this utility function, so we'd be interested to see which connection the activity is related to.